orchestra/todos/views.py

```diff
@@ -37,13 +40,35 @@ def update_todos_from_todolist_template(request):
         raise BadRequest('TodoList Template not found for the given slug.')
 
 
+@api_endpoint(methods=['GET'],
+              permissions=(permissions.IsAuthenticated,
+                           IsAssociatedWithProject),
+              logger=logger)
+def recommendations(request):
+    try:
+        todoQAs = TodoQA.objects.filter(
+            todo__task__assignments__worker__user=request.user, approved=False)
+        todos_recommendation = {todoQA.todo.description: TodoQASerializer(
```

Comment:
are we just basing things on overlapping descriptions for now? OK to say yes---just sanity-checking :)

Reply:
Yes. we are basing things on overlapping descriptions.